PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/CONTRIBUTING.MD

```diff
@@ -0,0 +1,15 @@
+### Configuring MS Application Insights
+- An Azure account with an active subscription.
```

Comment:
With documents like this one, we usually point people to the documentation on what an Azure account is and how to create one. Take [this section](https://github.com/Azure/azure-sdk-for-java/blob/3c094f92df0026e169f9014a8e71af3ca724244f/sdk/keyvault/azure-security-keyvault-keys/README.md?plain=1#L55) of the Key Vault README, for example.

Reply:
done -- made contributing.md more extensive here https://github.com/Njeriiii/azure-tools-for-java/pull/7